README.md

```diff
@@ -1,3 +1,11 @@
+# :warning: Moved to https://github.com/moby/moby/tree/master/libnetwork
```

Comment:
Maybe better to use the new warning blockquote? https://github.com/github-community/community/discussions/16925#discussion-4085374

Reply:
done